module/VuFind/tests/unit-tests/src/VuFindTest/Marc/MarcReaderTest.php

```diff
@@ -54,7 +54,19 @@ public function testMarcReader()
 
         // Test round-trips
         $reader = new \VuFind\Marc\MarcReader($reader->toFormat('MARCXML'));
+        $this->assertXmlStringEqualsXmlString($marc, $reader->toFormat('MARCXML'));
         $reader = new \VuFind\Marc\MarcReader($reader->toFormat('ISO2709'));
+        $this->assertXmlStringEqualsXmlString($marc, $reader->toFormat('MARCXML'));
+        $reader = new \VuFind\Marc\MarcReader($reader->toFormat('JSON'));
+        $this->assertXmlStringEqualsXmlString($marc, $reader->toFormat('MARCXML'));
+
+        // Verify JSON schema
+        $validator = new \Opis\JsonSchema\Validator();
```

Comment:
If the validator is only used in tests, should it be a dev dependency? Or do you plan to use it elsewhere (or is it already in use somewhere that I overlooked?)?

Reply:
It *is* a dev dependency (in `"require-dev"` of composer.json), or am I missing something?

Reply:
Oops, I just misread the file. My apologies!